Tests/ProjectDescriptionTests/ConfigTests.swift

```diff
@@ -39,6 +39,15 @@ final class ConfigTests: XCTestCase {
         XCTAssertCodable(config)
     }
 
+    func test_config_toJSON_WITH_swiftVersion() {
```

Comment:
Nitpick: `WITH` -> `with`

Reply:
I agree. I just wanted to be consistent with the rest of this file.

I will replace all occurrences of `WITH` in this file.